docs/rfcs/0000-template.md

```diff
@@ -0,0 +1,47 @@
+---
+layout: page
+title: <your title>
+permalink: /docs/rfcs/<file-name>
+---
+
+* RFC PR: [<PR #>](https://github.com/mozilla-mobile/android-components/pull/<#>)
+* Start date: YYYY-MM-DD (Day of proposal posting.)
+* End date: YYYY-MM-DD (Initial deadline for feedback, subject to blocking changes required. The proposal can be merged immediately after stakeholder approval.)
+* Stakeholders: <github-username>, <github-username>
```

Comment:
<github-username> doesn't show up in the fie view, is that expected?
<img width="416" alt="Screenshot 2024-02-28 at 10 21 48" src="https://github.com/mozilla-mobile/firefox-android/assets/1833156/a68e0af9-be7e-4a4d-922e-de969d6f9ecb">